### PR TITLE
set student pages responsive

### PR DESCRIPTION
**Pull request by wishwajith13**
set responsive and header

### CODE REVIEW

**Comment by JanithR-creator** [comment]
All the changes are described in commit massage.

File: frontend/app/pages/doctor/appointments/page.tsx [hunk 1]
```diff
@@ -1,11 +1,321 @@
...
+        "id": "93-265-4695",
+        "name": "Kassia Ruggiero",
+        "details": "Sed sagittis. Nam congue, risus semper porta volutpat, quam pede lobortis ligula, sit amet eleifend pede libero quis orci. Nullam molestie nibh in lectus.",
+        "time": "2:57 PM"
+    }, {
+        "id": "81-248-1156",
+        "name": "Genny Norgate",
+        "details": "In sagittis dui vel nisl. Duis ac nibh. Fusce lacus purus, aliquet at, feugiat non, pretium quis, lectus.\n\nSuspendisse potenti. In eleifend quam a odio. In hac habitasse platea dictumst.\n\nMaecenas ut massa quis augue luctus tincidunt. Nulla mollis molestie lorem. Quisque ut erat.",
+        "time": "10:28 AM"
+    }]
```
Comment (IsuruSH) [inline frontend/app/pages/doctor/appointments/page.tsx:258]: add sample data to frontend/data/doctor folder

**Review by IsuruSH — APPROVED** [review]

**Review by IsuruSH — COMMENTED** [review]
Review and updated testing

**Review by IsuruSH — COMMENTED** [review]
approved

**Pull request merged**